src/server/migrations/20190918172031-scenarios.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+exports.up = function(db) {
+    return db.runSql(
+        `
+CREATE OR REPLACE FUNCTION updated_at() RETURNS TRIGGER AS $$
+  BEGIN
+    IF (NEW != OLD) THEN
```

Comment:
When would the columns be different, such that you had to add and updated_at column? Is there a scenario where an item is created without an updated_at field?

Reply:
Currently we don't set the updated_at field when an item is created, so it's NULL until a scenario row is updated. Then if there are any changes to any of a scenario's fields, we'll record the time the change was made. This if-statement check is confirming that the new data in the column is actually different from the old data in the column. 

Let me know if that doesn't cover your question!

Reply:
It's the same `updated_at` trigger from bocoup API -- It's a plpgsql script that checks that the "NEW" content is different from the "OLD" content, and if it is, sets the NEW.updated_at = CURRENT_TIMESTAMP before it's saved to the DB -- effectively, `UPDATE table SET value='foo'` automtically also does `SET updated_at = CURRENT_TIMESTAMP` but only if the UPDATE actually changed something